meetings/2019-11-06.md

```diff
@@ -0,0 +1,92 @@
+﻿# Node.js Foundation Technical Steering Committee (TSC) Meeting 2019-11-06
+
+
+## Links
+
+
+* **Recording**: https://www.youtube.com/watch?v=XCX5qfuT8tw
+* **Minutes Google Doc**: https://github.com/nodejs/TSC/issues/773
+
+
+## Present
+
+
+* Beth Griggs @BethGriggs (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Franziska Hinkelmann @fhinkel (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Gabriel Schulhof @gabrielschulhof (TSC)
+* Gireesh Punathil @gireeshpunathil (TSC)
+* Matteo Collina @mcollina (TSC)
+* Sam Roberts @sam-github (TSC)
+* Tobias Nießen @tniessen (TSC)
+
+## Agenda
+
+### Announcements
+
+
+* Gabriel - welcoming @legendecas
+* Meeting times may have shifted for people due to return to standard time
+  * let me know if you think we should re-evaluate the meeting times

```

Comment:
Whos is "me" in this sentence? Is it @mhdawson or someone else?

Reply:
it is @mhdawson I can update.